source/components/the-component-lifecycle.md

```diff
@@ -116,6 +301,7 @@ Let's use this hook to cleanup our date picker and event listener from above:
 
 ```js
 willDestroyElement() {
+  this._super(...arguments);
```

Comment:
could you actually go through the guides in a separate PR and add the super calls?